board/toradex/apalis-imx8/apalis-imx8.c

```diff
@@ -268,7 +272,7 @@ int board_init(void)
 	return 0;
 }
 /* todo: With that function in ther is no console output in linux, drop for now */
-#if 0
+#ifdef CONFIG_SPL_BUILD
```

Comment:
We might want to fix the todo comment above to reflect the new adjustment (as it's not entirely dropped now):
- We release the UART in the SPL hand-off
- But don't release it do to bug in the u-boot proper hand off